pkgs/by-name/pl/platformsh/package.nix

```diff
@@ -1,42 +1,34 @@
-{ common-updater-scripts, curl, fetchFromGitHub, jq, lib, php, writeShellScript }:
+{ stdenvNoCC, lib, fetchurl }:
 
-php.buildComposerProject (finalAttrs: {
+stdenvNoCC.mkDerivation rec {
   pname = "platformsh";
-  version = "4.17.0";
-
-  src = fetchFromGitHub {
-    owner = "platformsh";
-    repo = "legacy-cli";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-8x7Fl1bYZIND4PuxVmPFNO2QOjeLMiIXh409DXG/WMU=";
-  };
-
-  vendorHash = "sha256-nXPfFlKYi2qP1bTeurRsopncKWg4zIZnZsSX/i0SF/s=";
-
-  prePatch = ''
-    substituteInPlace config-defaults.yaml \
-      --replace "@version-placeholder@" "${finalAttrs.version}"
-  '';
-
-  passthru.updateScript = writeShellScript "update-${finalAttrs.pname}" ''
-    set -o errexit
-    export PATH="${lib.makeBinPath [ curl jq common-updater-scripts ]}"
-    NEW_VERSION=$(curl -s https://api.github.com/repos/platformsh/legacy-cli/releases/latest | jq .tag_name --raw-output)
-
-    if [[ "v${finalAttrs.version}" = "$NEW_VERSION" ]]; then
-      echo "The new version same as the old version."
-      exit 0
-    fi
-
-    update-source-version "platformsh" "$NEW_VERSION"
+  version = "5.0.12";
+
+  src = {
```

Comment:
can you also add the macOS releases?

Reply:
I'm relatively new to nix and I'm unsure what you need here. Do you have an example or docs I could follow. This was originally a copy of the upsun package which is released under the same repository (different cli though)

Reply:
I was new to nixpkgs too when I did the upsun package, I think something like this https://github.com/NixOS/nixpkgs/pull/311643 should do it

Reply:
File has been updated to latest version with darwin too in https://github.com/nixos/nixpkgs/blob/51f5b20cdba6d80024fa5c76147146442eb8137c/pkgs/by-name/up/upsun/package.nix

Reply:
Thanks, I've replicated this.